releasenotes/notes/schedule-replace-2c5634a6133db237.yaml

```diff
@@ -0,0 +1,22 @@
+---
```

Comment:
this must be here in error ... fixup your git history

Reply:
This is because this based on the #4560. Waiting for this to merge.